hack/csi/manifests/vsphere/example-csi-vsphere.conf

```diff
@@ -0,0 +1,10 @@
+[Global]
+cluster-id = "<cluster-id>" # Grab with $ oc get clusterversion -o jsonpath='{.items[].spec.clusterID}{"\n"}'
+cluster-distribution = "Openshift"
+
+[VirtualCenter "vcenter.devqe.ibmc.devcluster.openshift.com"]
+insecure-flag = "true"
+user = "<username>"
+password = "<password>"
+port = "<port>" # Default is 443
+datacenters = "DEVQEdatacenter"
```

Comment:
What is this file for?

Reply:
It's a prereq for connecting the driver to vSphere. See: https://docs.vmware.com/en/VMware-vSphere-Container-Storage-Plug-in/3.0/vmware-vsphere-csp-getting-started/GUID-BFF39F1D-F70A-4360-ABC9-85BDAFBE8864.html

Reply:
That should be listed as `<datacenter>`.

Reply:
Does QE use DEVQEdatacenter too? If so, I think we should keep

Reply:
It depends who the audience is. We might be pointing customers at this too. So we should be generic. Same goes for the `[VirtualCenter]` section.

Reply:
Removed as this wasn't necessary as part of the CI steps to create a vSphere CSI volume and most of the secret data already exist elsewhere as part of the OCP install